config/routes.rb

```diff
@@ -1,5 +1,6 @@
 Rails.application.routes.draw do
   get 'home/index'
+  get 'study' => 'home#study'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.